packages/plugin-html-swipe-response/package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "@jspsych-contrib/plugin-html-swipe-response",
+  "private": "true",
+  "version": "1.0.0",
```

Comment:
The version should be 0.1 so that the changeset will auto bump to 1.0.

Reply:
Thanks! Resolved in [aae94fa](https://github.com/jspsych/jspsych-contrib/pull/36/commits/aae94fa279443a90d2768c8b33e3c5c69c781ec1)